automatic/datagrip/datagrip.nuspec

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>datagrip</id>
+    <version>0.0</version>
```

Comment:
this would have been fine to leave at the following version: `2017.1.1`, since that's the latest approved version available on chocolatey.org. (No need to change anything though)